src/components/pill/pill.spec.tsx

```diff
@@ -83,11 +93,13 @@ describe("Pill", () => {
 
     it("adds adds a click handler to the component", () => {
```

Comment:
Just a small typo - appreciate this is not part of your changes!
```suggestion
    it("adds a click handler to the component", () => {
```